lib/plugin.js

```diff
@@ -23,7 +24,13 @@ function plugin (pluginOptions) {
     }
 
     try {
-      extractSignature(req)
+      const sig0 = extractSignature(req)
+      const sig1 = constructSignatureString(req, options.sharedSecret)
+      console.log({
+        sig0,
+        sig1,
+        equal: sig0 === sig1
+      })
```

Comment:
I would find it easier to understand at a glance with more semantic variable names:

```suggestion
      const receivedSignature = extractSignature(req)
      const constructedSignature = constructSignatureString(req, options.sharedSecret)
      console.log({
        receivedSignature,
        constructedSignature,
        equal: receivedSignature === constructedSignature
      })
```

Reply:
Oh, yeah, this is really just for manual debugging. The short equal-character names `sig0` and `sig1` made for easy comparisons in the terminal.
This console stuff will all get ripped out before it's ready for review.